### DISCUSSION TITLE
How to uninstall a site from GUI?

### DISCUSSION THREAD

**Discussion by xuhdev** [filips123/PWAsForFirefox#299] [Q&A] [answered]
I installed a site as PWA. How do I uninstall it from a user interface?

**Comment by filips123** [comment, marked as answer]
You can click on the trash icon in the web app list in the extension:

![screenshot](https://user-images.githubusercontent.com/16626308/219080043-2bb25895-6abf-44ef-a134-ac1cac887eb6.png)

**Reply by KSXGitHub** [reply]
Huh? But there's no trash icon

**Reply by KSXGitHub** [reply]
![screenshot](https://github.com/filips123/PWAsForFirefox/assets/11488886/9b9da165-ec0a-48d6-98df-3c6b553e5e79)

**Reply by filips123** [reply]
You need to open the main extension's popup. It should be either pinned to the toolbar (not the address bar) or located inside the Firefox's extension menu.

![slika](https://github.com/filips123/PWAsForFirefox/assets/16626308/2cd900b7-8009-4a82-8391-1750f5c70992)

![slika](https://github.com/filips123/PWAsForFirefox/assets/16626308/a1089050-b7d0-40ae-9e46-6a9fed657760)
